Update ReactiveSwift to 5.0

mattprowse opened the pull request.

mattprowse committed:
Update dependencies and deployment targets

mattprowse committed:
Update to Swift 4.2

mattprowse committed:
Update SwiftPM and Travis Config

mdiep commented on ReactiveTask.xcodeproj/project.pbxproj:
```diff
@@ -389,7 +389,7 @@
 				CURRENT_PROJECT_VERSION = 1;
 				GCC_NO_COMMON_BLOCKS = YES;
-				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				MACOSX_DEPLOYMENT_TARGET = 10.10;
```
Instead of changing the deployment target for the project/framework, can you change it just for the test target?

mdiep reviewed (changes requested):
The podspec also needs to be updated.

mdiep commented:
Thanks for taking this on!

mattprowse committed:
Fix deployment targets

mattprowse committed:
Fix houndci warning

mattprowse committed:
Update Podspec

mattprowse commented:
No worries at all, I'm happy to help! I think I've addressed all of the PR comments. I'm not sure why `pod lib lint` is erroring on Travis though. It's succeeding for me locally.

mdiep commented:
Weird. I think we need to figure that out. Any ideas, @ikesyo?

ikesyo committed:
[CI] Update CocoaPods

ikesyo committed:
Remove target-level SWIFT_VERSION settings

ikesyo commented:
CocoaPods's version is the problem so updated in 291a87a.
